### DISCUSSION TITLE
Auto-Reconnect at X interval

### DISCUSSION THREAD

**Discussion by BradMcDev** [Chatterino/chatterino2#3246] [Ideas] [answered]
Hi all,

My partner and I run a 24/7 stream of our bunnies, and our viewers can redeem channel points for us to give them oats or fancy treats.

I run the stream off my computer, so I use chatterino to avoid having a popped out twitch chat all the time. It's great- thank you for the software! The only issue I have- the connection to channel points redemption feed disconnects SO often. I don't know if it's a twitch issue or a chatterino issue, but a simple fix in the meantime would be creating a "Auto-Reconnect every X minutes/hours" setting in the options, which would force a Ctrl+F5 reconnect to refresh the connection to the channel points redemptions.

It's frustrating because I don't know how quickly it disconnects from the channel points endpoint, but multiple times a day my partner has to tell me I missed a channel points redemption due to the disconnect.

An auto-reconnect feature would be incredibly helpful for people who interact with their channel points & run longer-than-average or 24/7 streams.

TY so much for considering!

**Comment by pajlada** [comment]
Which version are you on? In the latest unstable version (nightly) we've managed to make it disconnected a little bit less, so for now that'd be a decent alternative. https://wiki.chatterino.com/Help/#what-is-nightly-and-how-to-use-install-it

The bug where we don't reconnect properly can be tracked/followed further here: https://github.com/Chatterino/chatterino2/issues/1022

**Reply by BradMcDev** [reply]
Ahhh that's right, I forgot there was an open bug about the issue. Thanks for the suggestion on the unstable version, I'll give it a shot! Thanks!

**Comment by Felanbird** [comment, marked as answer]
Fixed by #3643
